web/wp-content/themes/lfevents/src/assets/scss/components/_forms.scss

```diff
@@ -1,3 +1,15 @@
 .grecaptcha-badge {
   visibility: hidden;
 }
+
+.hs-form-iframe {
+  background-color: white;
+}
```

Comment:
This is causing the background of the form to be white.  Why do we need this?  It looks odd when the form is in a group with a background:
<img width="463" alt="Screen Shot 2021-01-25 at 4 41 57 PM" src="https://user-images.githubusercontent.com/463160/105681584-4245c300-5f2c-11eb-87a6-063cf0839aa6.png">